ext/scheduler/airflow/__lib.py

```diff
@@ -470,23 +462,19 @@ def optimus_sla_miss_notify(dag, task_list, blocking_task_list, slas, blocking_t
         }
         # post event
         resp = optimus_client.notify_event(params["project_name"], params["namespace"], params["job_name"], event)
-        print("posted event ", params, event, resp)
+        log.info(f'posted event {params}, {event}, {resp}')
         return
     except Exception as e:
         print(e)
 
 def shouldSendSensorStartEvent(ctx):
     try:
-        ti = ctx.get('task_instance')
-        key = "sensorEvt/{}/{}/{}".format(ti.task_id , ctx.get('next_execution_date').strftime(TIMESTAMP_FORMAT) , ti.try_number)
-
-        result = ti.xcom_pull(key=key)
-        if not result:
-            print("sending NEW sensor start event for attempt number ", ti.try_number)
-            ti.xcom_push(key=key, value=True)
+        ti=ctx['ti']
```

Comment:
unnecessary variable.

Reply:
this was referenced in 2 places , so i created a variable for clarity